spec/controllers/booking_requests_controller_spec.rb

```diff
@@ -17,7 +17,7 @@
 
   let(:visitors_details) {
     {
-      email_address: 'ada@test.example.com',
+      email_address: 'ada@test.example',
```

Comment:
@KPobeeNorris why is the `.com` gone? It seems to me that `test.example.com` should pass the test.

Reply:
@StupidCodeFactory - thanks for the spot, I'll fix that now